### DISCUSSION TITLE
Example of a External Rotary Axis on a UR Robot

### DISCUSSION THREAD

**Discussion by MkMrA2** [visose/Robots#290] [Q&A] [answered]
Hi, 

I just started using the Robots plug-in. Thanks very much for creating it - I'm finding it pretty intuitive to get started. 

I'm using a UR10e robot. I want to add my own external rotary axis. Is it possible to get an example which shows how this can be accomplished? 

To better describe what I'm after, here's a screen grab from a Kuka|prc setup. The external axis is in front and the robot is printing on a rotating form. I can compute all the rotations of the external axis, and the back and forth motions of the robot. But I don't know how to configure this in Robots plug-in. 

![RotaryAxisExample](https://github.com/user-attachments/assets/466dcddd-8a82-4ccb-aeee-fc8a41c47239)

Is such a thing possible? 

Thanks!
Mark

**Comment by visose** [comment, marked as answer]
Hi @MkMrA2, external axes are not supported on UR robots.

**Reply by MkMrA2** [reply]
Okay, good to know. 

Mark
